src/tests/Microsoft.VisualStudio.Composition.Tests/AssembliesLazyLoadedTests.cs

```diff
@@ -270,87 +270,130 @@ internal void Initialize(Type cacheManagerType, Stream cachedComposition, Stream
                 this.container = containerFactory.CreateExportProvider();
             }
 
+            /// <summary>
+            /// Since the exceptions thrown by xUnit asserts are not serializable we lose information
+            /// in case tests fail in another appdomain. These four helpers throw a serializable exception
+            /// that results in a more information exception in the error log.
+            /// </summary>
+            internal void AssertFalse(bool condition, [CallerFilePath] string filePath = null, [CallerMemberName] string memberName = null, [CallerLineNumber] int lineNumber = 0)
```

Comment:
Can these methods be made static?
Can we move them to another class? (`AssertEx`, perhaps)